src/test/java/teammates/test/cases/webapi/UpdateFeedbackResponseActionTest.java

```diff
@@ -25,6 +27,16 @@
  */
 public class UpdateFeedbackResponseActionTest extends BaseActionTest<UpdateFeedbackResponseAction> {
 
+    private StudentAttributes student1InCourse1;
+    private StudentAttributes student2InCourse1;
+    private StudentAttributes student5InCourse1;
+    private InstructorAttributes instructor2OfCourse1;
+    private InstructorAttributes instructor1OfCourse1;
+    private FeedbackResponseAttributes student1ResponseToStudent1;
+    private FeedbackResponseAttributes testModerateResponse;
+    private FeedbackResponseAttributes student4ResponseToTeam;
```

Comment:
This is used in `testAccessControl_containsQuestionNotForInstructor_inaccessible` and `testAccessControl_studentAccessOtherTeam_inaccessible`, can `testAccessControl_containsQuestionNotForInstructor_inaccessible` use some other response?

Reply:
Hmm the assertion fails when I use `student1ResponseToStudent1` or `instructor1ResponseToAll`